pkg/reconciler/testing/table.go

```diff
@@ -156,12 +160,42 @@ func (r *TableRow) Test(t *testing.T, factory Factory) {
 			t.Errorf("Unexpected update (-want +got): %s", diff)
 		}
 	}
-	if got, want := len(actions.Updates), len(r.WantUpdates); got > want {
-		for _, extra := range actions.Updates[want:] {
+	if got, want := len(updates), len(r.WantUpdates); got > want {
+		for _, extra := range updates[want:] {
 			t.Errorf("Extra update: %#v", extra)
 		}
 	}
 
+	//TODO(dprotaso)Refactor
```

Comment:
Open an issue for this?

Reply:
https://github.com/knative/serving/issues/2843